src/dmd/root/stringtable.d

```diff
@@ -103,6 +103,17 @@ public:
         pools = null;
     }
 
+    /**
+    Looks up the given string in the string table and returns its associated
+    value.
+
+    Params:
+     s = the string to look up
+     length = the length of $(D_PARAM s)
```

Comment:
Technically `str` would need to be documented too (as it gets dittoed into this documentation.)

Reply:
We need a better way to inherit documentation that `ditto`. Fixed.